src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -518,6 +520,22 @@ ILoggingService InitializeLoggingService()
                 return loggingService;
             }
 
+            // VS builds discard many msbuild events so attach a binlogger to capture them all.
+            IEnumerable<ILogger> GetDebuggingLoggers()
+            {
+                if (CurrentProcessMatchesDebugName() is false ||
```

Comment:
Why do you need to say `is false`? Aren't there non-nullable bools?

Reply:
Just style preference, has nothing to do with nullability.

Reply:
I prefer !CurrentProcessMatchesDebugName, but that's fine. This is probably more readable, to be honest.